app/src/main/java/com/vinh/dictionary_1/data/source/local/DictTypeConverter.java

```diff
@@ -24,6 +27,9 @@ public static String fromBlob(byte[] blob){
 
     @TypeConverter
     public static byte[] fromString(String string) {
-        return string.getBytes();
+        if (string != null) {
+            return string.getBytes();
```

Comment:
ERROR: [I18N][DM_DEFAULT_ENCODING] Found reliance on default encoding in com.vinh.dictionary_1.data.source.local.DictTypeConverter.fromString(String): String.getBytes()
http://findbugs.sourceforge.net/bugDescriptions.html#DM_DEFAULT_ENCODING